bindata/network/ovn-kubernetes/microshift/clusterrole.yaml

```diff
@@ -59,6 +59,8 @@ rules:
   - get
   - list
   - watch
+  - egressservices
+  - egressservices/status
```

Comment:
This looks misplaced.

Reply:
:facepalm: Fixed!

Reply:
@oribon pointed out that this should be added for `openshift-ovn-kubernetes-node`
Updated the file.